internals/secrethub/rm_test.go

```diff
@@ -0,0 +1,270 @@
+package secrethub
+
+import (
+	"bytes"
+	"errors"
+	"testing"
+
+	"github.com/secrethub/secrethub-go/internals/assert"
+
+	"github.com/secrethub/secrethub-cli/internals/cli/ui/fakeui"
+	"github.com/secrethub/secrethub-go/internals/api"
+	"github.com/secrethub/secrethub-go/pkg/secrethub"
+	"github.com/secrethub/secrethub-go/pkg/secrethub/fakeclient"
+)
+
+func TestRmCommand_Run(t *testing.T) {
+
+	var testErr = errors.New("test")
+
+	cases := map[string]struct {
+		cmd                RmCommand
+		in                 string
+		argPath            api.Path
+		promptOut          string
+		out                string
+		err                error
+		promptError        error
+		deleteSecretError  error
+		deleteVersionError error
+		deleteDirError     error
+		newClientError     error
+		getTreeError       error
+		getSecretError     error
+	}{
+		"success-force-dir": {
```

Comment:
You can also have spaces in test case names, they will be replaced by underscores by `go test`. For the sake of consistency, I think we should stick to all lowercase, space separated test case names in code.

Reply:
This should now be solved! :)